src/main/java/org/jenkinsci/test/acceptance/controller/LocalController.java

```diff
@@ -131,19 +131,7 @@ public void postConstruct(Injector injector) {
     }
 
     private void installFormElementPath(File pluginDir) {
-        if (runInstallWizard) {
-            out.println("Skipping installation of form-element-path.jpi");
-        } else {
-            out.println("Installing form-element-path.jpi");
-            try {
-                FileUtils.copyFile(formElementPathPlugin, new File(pluginDir, "form-element-path.jpi"));
-            } catch (IOException e) {
-                String msg = String.format(
-                        "Failed to copy form-element-path file %s to plugin dir %s.", formElementPathPlugin, pluginDir
-                );
-                throw new RuntimeException(msg, e);
-            }
-        }
+        out.println("Skipping installation of form-element-path.jpi");
```

Comment:
I think before making non draft we need to retain backwards compatibility with Jenkins versions without the inlining (for the LTS) otherwise we won;t be able to run the ATH with new / updated plugin `PageObjects` against LTS versions, or we need to branch and maintain the compatibility.

Reply:
Yes of course 😄 . This was just to make sure it worked with the core form-element-path because I reworked the code to drop legacy, remove prototype.js and drop updating the dom every 1s.